### ISSUE TITLE
External storage quota

### ISSUE THREAD

**Issue by Ardinis** [nextcloud/server#4348]
### Steps to reproduce
1. Add a openstack swift external storage device
2. upload a content on the storage

### Expected behaviour
The system should counts the external storage in the usage quota 

### Actual behaviour
External storage is exclude from the calculation

### Server configuration

**Operating system**: Debian 8.0

**Web server:** apache2

**Database:** mariadb 10.0

**PHP version:** 5.6.30

**Nextcloud version:** (see Nextcloud admin page) 11.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:** fresh install

**Where did you install Nextcloud from:** nextcloud website

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
No errors have been found.

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** openstack swift

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** 

### Client configuration
**Browser:** Chrome

**Operating system:**

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
empty
```
</details>

**Comment by MariusBluem** [comment]
You now have a configurable option in config.php that controls whether external storage is counted against user’s quotas. This is still experimental, and may not work as expected. The default is to not count external storage as part of user storage quotas. If you prefer to include it, then change the default false to true:

'quota_include_external_storage' => false,

(See: https://docs.nextcloud.com/server/11/admin_manual/configuration_user/user_configuration.html#setting-storage-quotas)

**Comment by Ardinis** [comment]
Just as a note: it is still not counted with quota_include_external_storage enabled in config.php for openstack storage.

**Comment by MariusBluem** [comment]
@Ardinis And now, we have an issue 😜 Even if the feature is experimental 😶 @icewind1991

**Comment by MorrisJobke** [comment]
Fix of this is in #4816
